addon/components/rdf-input-fields/rich-text.hbs

```diff
@@ -0,0 +1,19 @@
+<AuLabel
+  @error={{this.hasErrors}}
+  @required={{this.isRequired}}
+  for={{this.inputId}}
+>
+  {{@field.label}}
+</AuLabel>
+
+{{#if @field.description}}
+  <p class="au-u-light au-u-margin-bottom-tiny">{{@field.description}}</p>
+{{/if}}
+
+<RdfaEditorContainer
```

Comment:
I would inline the component in the rich text component it doesn't provide that many benefits of splitting it up I think. Alternatively you could add it to a subfolder so it's clear that it's a "private" component? `<RdfInputFields::RichText::Editor>`

Reply:
The thing is that we are going to use it for other components, like now with the rdfa-editor component that includes plugins